### PR TITLE
service(mss): adds mobile-security-service manifest INTLY-3289

### PR DESCRIPTION
**Pull request by damienomurchu**
Add manifest for mobile-security-service

### CODE REVIEW

File: integreatly-mobile-security-service/0.4.1/mobile-security-service-operator.v0.4.1.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+      0 * * *"}}]'
+    capabilities: Basic Install
+    categories: Security
+    description: >-
+      This is the server component of the AeroGear Mobile Security Service. It is a RESTful API that allows developers to view, enable and disable specific versions of applications on demand, with the information stored in a PostgreSQL database.
+    containerImage: mobile-security-service-operator
```
Comment (philbrookes) [inline integreatly-mobile-security-service/0.4.1/mobile-security-service-operator.v0.4.1.clusterserviceversion.yaml]: This is only metadata, but should still point to a real image e.g. `quay.io/aerogear/mobile-security-service:v0.4.1`

File: integreatly-mobile-security-service/0.4.1/mobile-security-service-operator.v0.4.1.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                            fieldPath: metadata.namespace
+                      - name: APP_NAMESPACES
+                        value: mobile-security-service-apps
+                      - name: POD_NAME
+                        valueFrom:
+                          fieldRef:
+                            fieldPath: metadata.name
+                      - name: OPERATOR_NAME
+                        value: mobile-security-service-operator
+                    image: quay.io/aerogear/mobile-security-service-operator:master
```
Comment (philbrookes) [inline integreatly-mobile-security-service/0.4.1/mobile-security-service-operator.v0.4.1.clusterserviceversion.yaml]: it's better to point at a specific tag, as `master` could change in the future and break our install without us doing anything.

**Comment by damienomurchu** [comment]
@maleck13 changes requested by Phil made, and PR ready to be merged

**Comment by maleck13** [comment]
@damienomurchu great I have asked the mobile team to review their permissions.

**Comment by damienomurchu** [comment]
@maleck13 there's an issue to add the CSV to the MSS-operator repo, and I can do that now this is merged

cc @grdryn @wei-lee

**Pull request merged**